pkg/registry/apis/secret/contracts/secure_value.go

```diff
@@ -37,6 +38,7 @@ type SecureValueMetadataStorage interface {
 	SetVersionToActive(ctx context.Context, namespace xkube.Namespace, name string, version int64) error
 	SetVersionToInactive(ctx context.Context, namespace xkube.Namespace, name string, version int64) error
 	SetExternalID(ctx context.Context, namespace xkube.Namespace, name string, version int64, externalID ExternalID) error
+	MatchingOwner(ctx context.Context, namespace xkube.Namespace, ownerReference metav1.OwnerReference) (secureValueNames []string, err error)
```

Comment:
q: what happens if the save fails in unified storage? a: we delete a secure value that is actively being used :(